Library/ViewModels/FindFriendsFriendFollowCellViewModel.swift

```diff
@@ -135,6 +143,39 @@ public final class FindFriendsFriendFollowCellViewModel: FindFriendsFriendFollow
       )
       .skipRepeats()
 
+    self.followButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Follow_friend_name",
+          defaultValue: "Follow %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    self.unfollowButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Stop_following_friend_name",
+          defaultValue: "Stop following %{friend_name}",
```

Comment:
We should make this simply "Unfollow %{friend_name}". "Stop following" is closer to a `hint` than a `label`. If you wanted to add a hint, it would be "Stops following %{friend_name}." but that would be redundant if we have the label.